checkov/common/bridgecrew/vulnerability_scanning/image_scanner.py

```diff
@@ -109,5 +113,28 @@ def scan(self, docker_image_id: str, dockerfile_path: str, skip_extract_image_na
             logging.error("Failed to scan docker image", exc_info=True)
             return 1
 
+    def get_scan_results_from_cache(self, image_id: str) -> Dict[str, Any]:
+        image_id_encode = quote_plus(':'.join(['sha256', image_id]))
+        response = requests.request(
+            "GET", f"{bc_integration.api_url}/api/v1/vulnerabilities/scan-results/{image_id_encode}",
+            headers=bc_integration.get_default_headers("GET")
+        )
+        response.raise_for_status()
+        response_json = response.json()
+        output_type = response_json["outputType"]
+
+        if output_type == "Error":
+            logging.error(response_json["outputData"])
+            return {}
+
+        elif output_type == "Result":
+            return json.loads(
+                decompress_file_gzip_base64(
+                    response.json()["outputData"]
```

Comment:
```suggestion
                    response_json["outputData"]
```
no need to do it again